src/Diagnostics.RuntimeHost/Controllers/HostingEnvironment/HostingEnvironmentController.cs

```diff
@@ -72,6 +72,23 @@ public HostingEnvironmentController(IStampService stampService, ICompilerHostCli
             return await base.GetDetector(ase, detectorId, startTime, endTime, timeGrain);
         }
 
+        [HttpPost(UriElements.Detectors + UriElements.DetectorResource + UriElements.Statistics + UriElements.StatisticsResource)]
+        public async Task<IActionResult> GetSystemInvoker(string subscriptionId, string resourceGroupName, string hostingEnvironmentName, string invokerId, string detectorId, [FromBody] DiagnosticStampData postBody, string startTime = null, string endTime = null, string timeGrain = null)
+        {
+            if (postBody == null)
```

Comment:
I would say remove this PostBody from this API. We dont need to create a full resource for this API

Reply:
Sure I have removed that.